.github/workflows/general.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       #     key: sqlx
       - name: Run non-api tests
         run: |
-          cargo test --workspace --all-features --exclude api
+          cargo test --workspace --all-features --exclude api --jobs 1
```

Comment:
Why are jobs restricted to 1 here?

Reply:
https://github.com/supabase/pg_replicate/pull/78#issuecomment-2547597693